engines/sci/engine/kevent.cpp

```diff
@@ -78,6 +71,21 @@ reg_t kGetEvent(EngineState *s, int argc, reg_t *argv) {
 
 	curEvent = g_sci->getEventManager()->getSciEvent(mask);
 
+	// For Mac games with an icon bar, handle possible icon bar events first
+	if (g_sci->hasMacIconBar()) {
+		reg_t iconObj = NULL_REG;
+		if (g_sci->_gfxMacIconBar->handleEvents(curEvent, iconObj)) {
+			if (!iconObj.isNull()) {
+				invokeSelector(s, iconObj, SELECTOR(select), argc, argv, 0, NULL);
+			}
+
+			// The mouse press event was handled by the mac icon bar so change
+			// its type to none so that generic event processing can continue
+			// without the mouse press being handled twice
+			curEvent.type = SciEventType::kSciEventNone;
```

Comment:
Suggestion as per my other comment, change this to:
curEvent.type = kSciEventNone;

Reply:
@sluicebox : Yep, have tested locally with gcc-8.2.0 on Linux x86_64. This branch fails with the same error, but making this change fixes compilation.

Reply:
@digitall Thanks! I completely misread that error message in the exact same way a dozen times, never even looked at the right line of code once =)